paket.dependencies

```diff
@@ -2,17 +2,16 @@ framework: net45
 source https://www.nuget.org/api/v2
 
 nuget FSharp.Control.AsyncSeq
-nuget MongoDB.Bson ~> 2.0.0
-nuget MongoDB.Driver.Core ~> 2.0.0
+nuget MongoDB.Driver.Core
 
 // for building the project
 nuget FAKE
 nuget SourceLink.Fake
 
 // for executing tests
-nuget NUnit >= 2.6.2 < 3
-nuget NUnit.Runners >= 2.6.2 < 3
-nuget Unquote ~> 3
+nuget NUnit ~> 2
```

Comment:
ok, my intention was just to use most actual versions of Unquote and NUnit 2, or switch to NUnit3 later. revert?